core/arch/arm/cpu/neoverse-v2.mk

```diff
@@ -0,0 +1,3 @@
+arm64-platform-cpuarch 	:= neoverse-v2
```

Comment:
No arm32 support for EL0?

Reply:
Can you please explain more why would we need arm32 support here ?

Reply:
OP-TEE supports loading and executing both AArch64 and AArch32 TAs at S-EL0. I don't see a reason to stop it if the hardware supports it.

Reply:
Well it wouldn't hurt to add it.
Excuse me if it is obvious, but would it be achieved by simply adding the following ?

In neoverse-v2.mk
```
arm32-platform-cpuarch 	:= neoverse-v2
```

and in cortex-armv9.mk:
```
arm32-platform-cflags 	+= -mcpu=$(arm32-platform-cpuarch)
arm32-platform-aflags 	+= -mcpu=$(arm32-platform-cpuarch)
arm32-platform-cxxflags	+= -mcpu=$(arm32-platform-cpuarch)
```

Or is there something extra that needs to be done ?

Reply:
Looks good to me.

Reply:
Done